sdk/spring/azure-spring-boot-test-selenium-common/src/main/java/com/azure/test/aad/common/SeleniumITHelper.java

```diff
@@ -4,28 +4,51 @@
 package com.azure.test.aad.common;
 
 import com.azure.spring.test.AppRunner;
-import java.util.Map;
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
 import org.openqa.selenium.support.ui.WebDriverWait;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Map;
 
 public class SeleniumITHelper {
+    Logger logger = LoggerFactory.getLogger(SeleniumITHelper.class);
+    public static String folderName = "webdriver";
 
     protected AppRunner app;
     protected WebDriver driver;
     protected WebDriverWait wait;
 
-    public SeleniumITHelper(Class<?> appClass, Map<String, String> properties) {
-        createDriver();
+    public SeleniumITHelper(Class<?> appClass, Map<String, String> properties)  {
+        createDriver(appClass);
         createAppRunner(appClass, properties);
     }
 
-
-    protected void createDriver() {
+    protected void createDriver(Class<?> appClass) {
         if (driver == null) {
-            System.setProperty("wdm.cachePath", getClass().getClassLoader().getResource("selenium").getPath());
+            String currentPath = appClass.getClassLoader().getResource(("")).getPath();
+            String sdkSpring = File.separator + "sdk" + File.separator + "spring";
+            String destination = currentPath + File.separator + folderName;
```

Comment:
Rename `destination` to `webDriverCachePath` will make code easier to understand.